data/build_movielens.py

```diff
@@ -83,25 +88,87 @@ def main(_):
                          tqdm.tqdm(user_seqs_ids)))
 
     def format_sequence(arr):
-      """map a list of movie names to a tab-separated string
-      example:
+      """Format a single movie list into a set of training examples.
+
+      Args:
+        arr: athe array of 10 movies in a single user sequence
+
+      Returns:
+        a list of strings containing formatted sequence examples
       """
-      inputs = "@ %s @" % " @ ".join(arr[:-1])
-      targets = arr[-1]
-      return "\t".join((inputs, targets))
+      random.shu
```

Comment:
I switched from using a function to just formatting the sequences in a for loop, because the map_reduce function was making it take much longer. This function isn't being called anymore so I removed it (that's why the hanging line didn't cause any errors).